packages/cfd/src/Containers/cfd-dbvi-onboarding.tsx

```diff
@@ -1,88 +1,150 @@
 import React from 'react';
-import { DesktopWrapper, MobileDialog, MobileWrapper, Modal, UILoader, Loading } from '@deriv/components';
+import {
+    Button,
+    DesktopWrapper,
+    Icon,
+    Loading,
+    MobileDialog,
+    MobileWrapper,
+    Modal,
+    Text,
+    UILoader,
+} from '@deriv/components';
 import { localize } from '@deriv/translations';
 import RootStore from 'Stores/index';
 import { PoiPoaDocsSubmitted } from '@deriv/account';
 import { connect } from 'Stores/connect';
-import { WS, getAuthenticationStatusInfo } from '@deriv/shared';
-import { AccountStatusResponse, GetAccountStatus } from '@deriv/api-types';
+import { getAuthenticationStatusInfo, isMobile, WS } from '@deriv/shared';
+import { AccountStatusResponse } from '@deriv/api-types';
+import { TCFDDbViOnBoardingProps } from './props.types';
 import CFDFinancialStpRealAccountSignup from './cfd-financial-stp-real-account-signup';
 
-type TVerificationModalProps = {
-    disableApp: () => void;
-    enableApp: () => void;
-    is_cfd_verification_modal_visible: boolean;
-    toggleCFDVerificationModal: () => void;
-    jurisdiction_selected_shortcode: string;
-    updateAccountStatus: () => void;
-    account_status: GetAccountStatus;
+type TSwitchToRealAccountMessage = {
+    onClickOK: () => void;
 };
 
+const SwitchToRealAccountMessage = ({ onClickOK }: TSwitchToRealAccountMessage) => {
+    return (
+        <div className='da-icon-with-message'>
+            <Icon icon={'IcPoaLock'} size={128} />
+            <Text
+                className='da-icon-with-message__text'
+                as='p'
+                color='general'
+                size={isMobile() ? 'xs' : 's'}
+                line_height='m'
+                weight='bold'
```

Comment:
Remove default values.
```suggestion
            <Text
                className='da-icon-with-message__text'
                as='p'
                size={isMobile() ? 'xs' : 's'}
                weight='bold'
```